<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>617</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>52</red>
        <green>101</green>
        <blue>164</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>78</red>
        <green>152</green>
        <blue>246</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>65</red>
        <green>126</green>
        <blue>205</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>26</red>
        <green>50</green>
        <blue>82</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>67</green>
        <blue>109</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>52</red>
        <green>101</green>
        <blue>164</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>153</red>
        <green>178</green>
        <blue>209</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>52</red>
        <green>101</green>
        <blue>164</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>78</red>
        <green>152</green>
        <blue>246</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>65</red>
        <green>126</green>
        <blue>205</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>26</red>
        <green>50</green>
        <blue>82</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>67</green>
        <blue>109</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>52</red>
        <green>101</green>
        <blue>164</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>153</red>
        <green>178</green>
        <blue>209</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>26</red>
        <green>50</green>
        <blue>82</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>52</red>
        <green>101</green>
        <blue>164</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>78</red>
        <green>152</green>
        <blue>246</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>65</red>
        <green>126</green>
        <blue>205</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>26</red>
        <green>50</green>
        <blue>82</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>67</green>
        <blue>109</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>26</red>
        <green>50</green>
        <blue>82</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>26</red>
        <green>50</green>
        <blue>82</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>52</red>
        <green>101</green>
        <blue>164</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>52</red>
        <green>101</green>
        <blue>164</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>52</red>
        <green>101</green>
        <blue>164</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabPages">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>591</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabMessage">
     <attribute name="title">
      <string>Messages</string>
     </attribute>
     <widget class="QLineEdit" name="txtMsg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>511</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSend">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>450</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>SEND</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lstMsgHistory">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>581</width>
        <height>441</height>
       </rect>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabUpdate">
     <attribute name="title">
      <string>Update User</string>
     </attribute>
     <widget class="QLabel" name="imgPic">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="pixmap">
       <pixmap>Resources/defaultuser.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblUser">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblName">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtUser">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>50</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtName">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>80</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUpdatePic">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Profile Picture</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPwd">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>250</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSave">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>250</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLogout">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>250</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabView">
     <attribute name="title">
      <string>View User</string>
     </attribute>
     <widget class="QLineEdit" name="txtName_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>80</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblName_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtUser_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>50</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="imgPic_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="pixmap">
       <pixmap>Resources/defaultuser.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblUser_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSearch">
     <attribute name="title">
      <string>Search</string>
     </attribute>
     <widget class="QTableWidget" name="tblResults">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>591</width>
        <height>441</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Picture</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Username</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btnAddContact">
      <property name="geometry">
       <rect>
        <x>458</x>
        <y>450</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Contact</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QListWidget" name="lstContact">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>501</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::SelectedClicked</set>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="resizeMode">
     <enum>QListView::Adjust</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSearch">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>511</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGroup">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>520</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Group Chat</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
